lib/buildkite/test_collector/rspec_plugin/reporter.rb

```diff
@@ -34,6 +35,13 @@ def dump_summary(notification)
 
         Buildkite::TestCollector.session.close(examples_count)
       end
+
+      Buildkite::TestCollector.session.add_example_to_send_queue(example.id)
+    end
+
+    def dump_summary(_)
```

Comment:
I can't find the correct search term for this, why do you pass an underscore as an argument here?

Reply:
There's a convention in Ruby where unused variables are prefixed with underscore, or just stored as underscore. As `dump_summary` is an alias of three methods that rspec expects to call with an argument, I've just stored the argument here as `_` to indicate its unused. I could also have done `_notification` or something similar. Using just underscore itself is [slightly magic](https://po-ru.com/2012/09/21/rubys-magic-underscore) cos you can do it for multiple arguments. 

I think we also have a [linter](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/UnusedMethodArgument) for this in buildkite/buildkite but not on this repo.

Reply:
Lol jinx @gchan I didn't see your comment. I can't see where you're seeing two `dump_summary` methods?

Reply:
Oh yep, I'm familiar with the `_` syntax, but was thrown by the lack of a argument name. +1 to @gchan's suggestion for readability